templates/topic/reports/a11y-1.json

```diff
@@ -0,0 +1,689 @@
+      "acr:relevancy": "acr:forEvaluation",
+      "dct:description": "",
+      "earl:outcome": "earl:passed",
+      "selected": "act:rulesets/wcag2x/sc/non-text-content#situations"
+    },
+    {
+      "earl:test": "WCAG21:audio-only-and-video-only-prerecorded",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:notRelevant",
+      "acr:relevancy": "acr:notRelevant",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable",
+      "selected": "act:rulesets/wcag2x/sc/non-text-content#situations"
+    },
+    {
+      "earl:test": "WCAG21:captions-prerecorded",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:notRelevant",
+      "acr:relevancy": "acr:notRelevant",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable",
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:audio-description-or-media-alternative-prerecorded",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:notRelevant",
+      "acr:relevancy": "acr:notRelevant",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable",
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:captions-live",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:notRelevant",
+      "acr:relevancy": "acr:notRelevant",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable",
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:audio-description-prerecorded",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:notRelevant",
+      "acr:relevancy": "acr:notRelevant",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable",
+      "selected": ""
+    },
+    {
+      "earl:test": "WCAG21:info-and-relationships",
+      "earl:subject": {
+        "@id": "_info-and-relationships_itm1",
+        "dct:isPartOf": "_:subject",
+        "@type": ["earl:TestSubject", "schema:WebPageElement"],
+        "dct:description": "The page provides semantic structure, including headings and lists, to make information and relationships clear and accessible."
+      },
+      "earl:result": [
+        {
+          "earl:test": "act:rules/wcag2x/sc/1_3_1_situation_a",
+          "earl:outcome": "earl:passed",
+          "earl:mode": "earl:manual",
+          "earl:subject": "_info-and-relationships_itm1",
+          "dct:description": "h1,h2, and h3 tags are used to identify headings and subheadings, respectively."
+        },
+        {
+          "earl:test": "act:rules/wcag2x/sc/1_3_1_situation_a",
```

Comment:
humm, the test ID are the same as the preceding (line 128). Should be a reference (URL) to the applicable technique?

Reply:
If I wanted to reference the following technique ([H48: Using ol, ul and dl for lists or groups of links](https://www.w3.org/WAI/WCAG22/Techniques/html/H48.html)), how would I do so?

Reply:
@BrahimMahadi Voici comment

```
"earl:test": "WCAG2xTech:html/H48",
```

Tu peux voir toute ces URL ainsi que leur préfix en explorant le fichier JSON-LD passé en @context. Ici c'est ce fichier : https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld 

Voici la version du context mais via son emplacement code source: https://github.com/wet-boew/vocab/blob/main/src/context/2023/assessment-report-en.json#L143  

Les préfix sont des abrégé d'IRI, similar à une URL

Donc cet IRI: WCAG2xTech:html/H48
Le préfix "WCAG2xTech" sera résolue par : https://www.w3.org/WAI/WCAG21/Techniques/
Et l'IRI equivalent expanded est : https://www.w3.org/WAI/WCAG21/Techniques/html/H48 

De manière alternative, tu peux utiliser l'URL du document de la technique du WCAG. Cela devrait résoudre au même IRI une fois expanded.

cc @Garneauma

Reply:
Comments addressed